runtime/profile/_/mod/core/data/god.lua

```diff
@@ -1,33 +1,226 @@
+local function secret_treasure(id)
+   return { id = "core.secret_treasure", param1 = id, nostack = true }
```

Comment:
The parameter of `param1` does not affect the result. All secret treasure which gods give you become a just "secret treasure" with `param1=0`.